### PR TITLE
added more tests

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: test/ServerRequestTest.php [hunk 1]
```diff
@@ -97,7 +97,19 @@ public function testRemovingAttributeReturnsCloneWithoutAttribute($request)
         $this->assertNull($new->getAttribute('foo', null));
     }
 
-    public function testUsesProvidedConstructorArguments()
+    public function provideMethods()
+    {
+        return array(
```
Comment (bakura10) [inline test/ServerRequestTest.php]: Use short array notation (Diactoros is PHP 5.5+)

Comment (weierophinney) [inline test/ServerRequestTest.php]: Agreed; that's one thing I've enjoyed about developing on phly/http and now diactoros. :)

Comment (samsonasik) [inline test/ServerRequestTest.php]: done ;)

Warm regards,

Abdul Malik Ikhsan

Pada 22 Mei 2015, pukul 21.52, weierophinney <notifications@github.com> menulis:

Comment (Maks3w) [inline test/ServerRequestTest.php]: Please avoid review comments about styling. If the CS tool not yield there is nothing to discuss.

**Pull request merged**